app/controllers/teams_controller.rb

```diff
@@ -31,6 +31,24 @@ def create
     end
   end
 
+  # PATCH/PUT /teams/1
+  # PATCH/PUT /teams/1.json
+  def update
```

Comment:
This is *identical* to the one from the namespaces_controller. Maybe we could merge functionality through a concern.